scripts/release/issue-list.sh

```diff
@@ -17,9 +17,10 @@
 programname=$0
 function usage()
 {
-    echo "usage: $programname -r repository -m milestone"
+    echo "usage: $programname -r repository -m milestone -f format"
     echo "  -r  repository name"
     echo "  -m  milestone id"
+    echo "  -f  output format ('human' or 'jsonstring')"
```

Comment:
Why do we need that? And even if we need it, what 'jsonstring' produces is not JSON. This is about whether new line characters are printed or now. What is the driving force here for this?